go/vt/vttablet/tabletmanager/semisyncmonitor/monitor.go

```diff
@@ -416,3 +437,46 @@ func (m *Monitor) addWaiter() chan struct{} {
 func (m *Monitor) bindSideCarDBName(query string) string {
 	return sqlparser.BuildParsedQuery(query, sidecar.GetIdentifier()).Query
 }
+
+func (m *Monitor) addLockTimeout(query string) string {
+	timeoutQuery := fmt.Sprintf(setTimeoutQuery, int(m.actionTimeout.Seconds()))
+	return fmt.Sprintf("%s;%s", timeoutQuery, query)
```

Comment:
I think the new sprintf performance linter might complain about this.

Reply:
@arthurschreiber I would have expected `return timeoutQuery + ";" + query` too 🤔 

You can see what the linter's auto-fix is with `golangci-lint run --fix ./go/...`

Reply:
It's more efficient if you're doing a single append w/o any transformations needed, but otherwise no AFAIUI. There's nothing magical about appends, it creates a new string that consists of the contents of the two.

If we care (which we do not IMO as this is very low concurrency) we can use StringBuilder.